setup.py

```diff
@@ -292,7 +292,9 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'google-cloud-kms>=2.0.0,<3.0.0',
     'google-cloud-language>=1.1.1,<2.0.0',
     'google-cloud-logging>=2.1.1,<3.0.0',
-    'google-cloud-memcache>=0.2.0',
+    # 1.1.0 removed field_mask and broke import for released providers
+    # We can remove the <1.1.0 limitation after we release new Google Provider
+    'google-cloud-memcache>=0.2.0,<1.1.0',
```

Comment:
Are you sure `google-cloud-memcache v1.1.0` is compatible? These two versions were released within a short period of time.
> 1.1.0 (2021-06-30)
> 1.0.0 (2021-05-28)

Reply:
The new version may have some compatibility, but not enough for the operator to be successfully executed.

Reply:
Pretty sure 1.1.0 was released yesterday (from pypi):

![Screenshot 2021-07-13 20 42 53](https://user-images.githubusercontent.com/595491/125507675-eed90385-d8b2-40ec-8e11-1cce3618ffcd.png)